New initiative for Key Vault guardrails and Corp Networking (#1213)
Azure/Enterprise-Scale: docs/wiki/Whats-new.md

    - [Deny vNet peering cross subscription.](https://www.azadvertizer.net/azpolicyadvertizer/Deny-VNET-Peer-Cross-Sub.html)
    - Portal accelerator updated with additional tab and options to enable this initiative.
- Added assignment of "Network interfaces should not have public IPs" built-in Policy to the 'Corp' Management Group.
- Added a policy assignment to deny Classic resources at the intermediate root management group
- Added new initiative and assignment to implement recommended guardrails for Azure Key Vault at the landing zones management group
  - Initiative name: ENFORCE-Guardrails-KeyVault
  - Policies included: [ALZ Polices](https://aka.ms/alz/policies)
  - Portal accelerator updated
- Added two new policy assignments to govern Corp Management Group networking:

>   - DENY-AlzCorpNetworking - blocks the provisioning onf vWAN/ER/VPN gateways in Corp

Reviewer: Should we rename this to something more specific. eg. `Deny-HybridNetworking` as its a bit generic ATM and could lead to "panic" by consumers not understanding it without digging into it

>   - AUDIT-AlzCorpNetworking - audits the provisioning of Private Link / Private DNS Zones in Corp

Reviewer: See comment below, this policy needs some re-work. But also we should rename to something link `Audit-PeDnsZones`

    - [Deny vNet peering cross subscription.](https://www.azadvertizer.net/azpolicyadvertizer/Deny-VNET-Peer-Cross-Sub.html)
    - Portal accelerator updated with additional tab and options to enable this initiative.
- Added assignment of "Network interfaces should not have public IPs" built-in Policy to the 'Corp' Management Group.
- Added a policy assignment to deny Classic resources at the intermediate root management group
- Added new initiative and assignment to implement recommended guardrails for Azure Key Vault at the landing zones management group

>   - Initiative name: ENFORCE-Guardrails-KeyVault

Reviewer: ```suggestion
  - Initiative name: `ENFORCE-Guardrails-KeyVault`
```

  - Policies included: [ALZ Polices](https://aka.ms/alz/policies)
  - Portal accelerator updated
- Added two new policy assignments to govern Corp Management Group networking:

>   - DENY-HybridNetworking - blocks the provisioning onf vWAN/ER/VPN gateways in Corp

Reviewer: ```suggestion
  - `DENY-HybridNetworking` - blocks the provisioning of vWAN/ER/VPN, including gateways, in Corp
```

>   - AUDIT-PeDnsZones - audits the provisioning of Private Link Private DNS Zones in Corp

Reviewer: ```suggestion
  - `AUDIT-PeDnsZones` - audits the provisioning of Private Link Private DNS Zones in Corp
```

>     - NOTE: The policy default values include all the static Private DNS Zones only. The ALZ assignment includes dynamic assignments for Azure Backup, AKS, Kusto and Batch.

Reviewer: ```suggestion
    - **NOTE**: The policy default values include all the static Private DNS Zones only. When assigned via the ALZ portal experience the assignment includes all the Private DNS Zones that are deployed as part of the ALZ Portal experience, including the geo code/regional zones for Azure Backup, AKS etc. 
```

> - Added new policy assignment to audit WAF enabled on Application Gateways (Audit-AppGW-WAF)

Reviewer: ```suggestion
- Added new policy assignment to audit WAF enabled on Application Gateways (`Audit-AppGW-WAF`)
```

> - Added new intiative and assignment to enable Azure Compute Security Baseline compliance auditing for Windows and Linux virtual machines (Enforce-ACSB)

Reviewer: ```suggestion
- Added new initiative and assignment to enable Azure Compute Security Baseline compliance auditing for Windows and Linux virtual machines (`Enforce-ACSB`)
```

> - Updated the initiative Deny-PublicPaaSEndpoints to include additional policies available to block public access for PaaS services

Reviewer: ```suggestion
- Updated the initiative `Deny-PublicPaaSEndpoints` to include additional policies available to block public access for PaaS services
```
